random/2016-07-ipfs-workshop-post.md

```diff
@@ -0,0 +1,21 @@
+Subject: IPFS Q3 Workshop
+
```

Comment:
would be useful to add another more high level paragraph. something that mentions:
- the IPFS project has gathered a lot of momentum in the last few months
- lots of exciting things are going on (maybe links?)
- there is a lot to demo
- there is a lot to discuss as a community

(i.e. make the case for "tons of awesome things happening, we need a sync workshop to talk about it")